modules/understanding-upgrade-channels.adoc

```diff
@@ -7,17 +7,156 @@
+performed well while available as a release candidate in the candidate-{product-version}
+channel from where they were promoted. Some time after a release appears in
+fast-{product-version}, and therefore after the release has had further use, it will
+also appear in stable-{product-version}. Releases will never appear in stable-{product-version}
+before they appear in fast-{product-version}.
+
+The fast-{product-version} channel also allows upgrading from a previous minor version of
+{product-title}.
+
+= stable-{product-version}
+
+Like the fast-{product-version} channel, the stable-{product-version} channel will
+only contain releases that have been declared general availability are therefore
+fully supported. However the stable-{product-version} channel will gradually roll out
+releases to customers based on data from our SRE teams, support services, and
+pre-production and production environments that participate in our connected customer
+program, rather than being immediately available as they are in the fast-{product-version}
+channel. For patch and CVE fixes this delay can range from several hours to a day
+and allows an extra period of assessment in how the software performs.
+
+The stable-{product-version} channel also allows upgrading from a previous minor version of
+{product-title}.
+
+= Upgrade Version Paths
+
+{product-title} maintains an upgrade recommendation service that understands the
+version of {product-title} you have installed as well as the path to take within
+the channel you choose to get you to the next release. You can imagine seeing the
+following in the fast-{product-version} channel:
+
+.  {product-version}.0
+.  {product-version}.1
+.  {product-version}.3
+.  {product-version}.4
+
+The service only recommends upgrades that have been tested and have no serious issues.
+If you are on {product-version}.1 and {product-title} is allowing you to select
+{product-version}.4, then it is safe for you to go from .{product-version}.1 to .{product-version}.4. Likewise,
+the absence of {product-version}.2 may be due to a CVE that was fixed in {product-version}.3 and Red Hat no
+longer suggests upgrading to a known vulnerable version.
+
+Update stability depends on your channel. The presence of an update recommendation in
+the candidate-{product-version} channel does not imply that the update is supported.
+It means that no serious issues have been found with the update yet, but there may
+not be significant traffic through the update to suggest stability. The presence of
+an update recommendation in the fast-{product-version} or stable-{product-version}
+channels is a declaration that the update is fully supported while it is in the
+channel. Like releases in the stable-{product-version} channel, update
+recommendations in the stable-{product-version} channel will be promoted from
+fast-{product-version} with a delay that can range from several hours to a day.
```

Comment:
I think this sentence can be dropped, since it's covered above in the stable-{product-version} section and doesn't speak to the stability of releases or update recommendations.

Reply:
This one?
"Like releases in the stable-{product-version} channel, update
recommendations in the stable-{product-version} channel will be promoted from
fast-{product-version} with a delay that can range from several hours to a day."

Reply:
> This one?

Yup.

Reply:
Removed sentence